regimes/co/extensions.go

```diff
@@ -8,8 +8,9 @@ import (
 
 // Extension keys used in Colombia.
 const (
-	ExtKeyDIANMunicipality cbc.Key = "co-dian-municipality"
-	ExtKeyDIANCorrection   cbc.Key = "co-dian-correction"
+	ExtKeyDIANMunicipality     cbc.Key = "co-dian-municipality"
+	ExtKeyDIANCorrectionCredit cbc.Key = "co-dian-correction"
```

Comment:
We need to separated correction extensions now, one for credit notes and another one for debit notes. I was shy to rename `co-dian-correction` to `co-dian-correction-credit`, but it's probably ok, nothin will break if we do that. @samlown WDYT?

Reply:
oh, that's very annoying... Do you know what the underlying code is called? This key was written some time ago, I'd be more tempted now to use a directly translation of the XML field name.

Reply:
These are the docs regarding those fields:

![image](https://github.com/user-attachments/assets/10a6a160-a8e8-44c0-8b1e-f35767094abf)

In both cases, the same UBL field is used, but the list of allowed values and their meaning are different depending on whether the type is a debit or a credit note. That's why I split the extensions in two.

Reply:
I think I'd be tempted to go with simply:
* `co-dian-credit-code`
* `co-dian-debit-code`
Reasoning: "correction" is implied in "credit" and "debit", and "code" makes it clear that this is something defined in a list of codes somewhere.

Reply:
Updating!

Reply:
Address in [88db17e](https://github.com/invopop/gobl/pull/323/commits/88db17e75e12fb4bd503cb91cf4cefa0763fafb8)